R/R/namespace.R

```diff
@@ -0,0 +1,24 @@
+#' Switch to a namespace specified by the given tag.
```

Comment:
We would need similar functionality for `metadata`

Reply:
Yep adding it now.